src/wasm/wasm-type.cpp

```diff
@@ -529,6 +531,15 @@ HeapType Type::getHeapType() const {
     }
     WASM_UNREACHABLE("Unexpected type");
   }
+  if (isRtt()) {
+    return getRtt().heapType;
+  }
+  WASM_UNREACHABLE("unexpected type");
+}
+
+const Rtt& Type::getRtt() const {
```

Comment:
Our (brand new) convention is to pass `Rtt`s by value rather than reference (but I'm happy to reconsider that).